docs/pages/development-guides/backend/login/login.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Login Backend
+has_parent: true
+parent: Backend
+grand_parent: Development Guides
+permalink: /pages/development-guides/backend/login
+nav_order: 100
+---
+
+# Login Backend
+
+- [Login Backend](#login-backend)
+  - [Build & Run](#build--run)
+    - [Prequest](#prequest)
+    - [Build](#build)
+    - [Run](#run)
+  - [Authentication flow](#authentication-flow)
+
+> This document refers to the code under `Phonebook.Backend\Phonebook.Backend.Login`
+
+## Build & Run
+
+### Prequest
+
+To build & run this backend you need the dotnet core SDK installed in version 3.1 or higher. The latest version can be found [here](dot.net).
+Also you need VSCode. I also recommend to install the [C# Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) from Microsoft. This helps with IntelliSense and live error checking. Also I recommand the use of [IntelliCode](https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.vscodeintellicode), this helps to write better code.
+
+### Build
+
+In vscode simply run the task `build Backend.Login`. If you want build this with the command line you must navigate to the `Phonebook.Backend\Phonebook.Backend.Login` and run the command `dotnet build`.
+
+### Run
+
+Before you can run the login backend the first time you must set some secrets. These secrets describe the connection and the trust between the identity provider and the login server. (See more in the [flow documentation](#authentication-flow))
```

Comment:
for the first time,

Reply:
use the github suggestion feature. It makes it a lot easier!
![image](https://user-images.githubusercontent.com/13590797/70235534-6c09bc80-1763-11ea-9d7f-9c6183a5ff5a.png)